pkgs/tools/misc/tewisay/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "tewisay-unstable-${version}";
```

Comment:
Could you remove "unstable"?

Reply:
Sorry, that's good like this in fact... (https://nixos.org/nixpkgs/manual/#sec-package-naming)